src/accountManager.ts

```diff
@@ -18,7 +18,7 @@ export class AccountManager {
     billingContract: BillingContract,
     slack: Slack,
     bondThreshold: bigint = TEN_ETH,
-    scanUrl?: string,
+    scanUrl?: string
```

Comment:
The lost trailing commas are the reason for failing CI. Perhaps your IDEs "formatOnSave" is configured differently than the lint config here.